go.mod

```diff
@@ -1,6 +1,6 @@
 module chainguard.dev/apko
 
-go 1.23.4
+go 1.24.0
```

Comment:
sure we want that?
this says that the min go is 1.24, and we force all to upgrade to go1.24

i am fine with that, just checking